### ISSUE TITLE
Add LightTable plugin

### ISSUE THREAD

**Issue by treyhunner** [editorconfig/editorconfig#212]
@cmstead mentioned this on Twitter: https://twitter.com/cm_stead/status/581241956002578432

I believe the JavaScript core library could be used for this.

**Comment by cmstead** [comment]
Yep! That's exactly what I was looking at. :-) I have the plugin underway, however, I'm kind of stuck so I am working on figuring out how to capture the workspace or file path from within a plugin.if anyone has ideas, I'm open to hearing them please have a look here for my progress on this:

https://github.com/cmstead/lighttable-editorconfig
